### PR TITLE
Updated index.html templates

### PR DESCRIPTION
**Pull request by thelovekesh**
Added button for ask question in `index.html` template

image

### CODE REVIEW

File: wordpress-20-04/files/var/www/html/index.html [hunk 1]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
+      .content_min {
```
Comment (bradford-hamilton) [inline wordpress-20-04/files/var/www/html/index.html:37]: nit: add space

File: lamp-18-04/files/var/www/html/index.html [hunk 2]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
+      .content_min {
```
Comment (bradford-hamilton) [inline lamp-18-04/files/var/www/html/index.html:37]: nit: add space

File: lamp-20-04/files/var/www/html/index.html [hunk 3]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
+      .content_min {
```
Comment (bradford-hamilton) [inline lamp-20-04/files/var/www/html/index.html:37]: nit: add space

File: lemp-18-04/files/var/www/html/index.html [hunk 4]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
```
Comment (bradford-hamilton) [inline lemp-18-04/files/var/www/html/index.html:36]: nit: add space

File: lemp-20-04/files/var/www/html/index.html [hunk 5]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
+      .content_min {
```
Comment (bradford-hamilton) [inline lemp-20-04/files/var/www/html/index.html:37]: nit: add space

File: mysql-20-04/files/var/www/html/index.html [hunk 6]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
+      .content_min {
```
Comment (bradford-hamilton) [inline mysql-20-04/files/var/www/html/index.html:37]: nit: add space

File: wordpress-18-04/files/var/www/html/index.html [hunk 7]
```diff
@@ -32,7 +32,19 @@
         margin: 32px auto;
         padding: 32px;
         text-align: center;
-        width: 960px;;
+        width: 960px;
+      }
+      .content_min {
```
Comment (bradford-hamilton) [inline wordpress-18-04/files/var/www/html/index.html:37]: nit: add space

**Review by bradford-hamilton — COMMENTED** [review]
Hey, thanks so much for the help! This looks great - I left a very small nit around spacing, and then we will be good to go!

**Comment by bradford-hamilton** [comment]
@thelovekesh beautiful, thanks!

**Comment by thelovekesh** [comment]
Always welcome @bradford-hamilton. You are the best reviewer and maintainer I have ever worked with <3

**Pull request merged**